js/run.js

```diff
@@ -254,7 +254,7 @@ var MD5 = function (string) {
     return temp.toLowerCase();
 }
 
-String.prototype.nl2br = function()
+var nl2br = function()
```

Comment:
Ah, PHP style naming, my favorite.

Maybe we should start enforcing Phobos style on the site JS.